backend/tarsy/integrations/llm/client.py

```
@@ -22,7 +22,7 @@
 
 from tarsy.config.settings import Settings
 from tarsy.hooks.typed_context import llm_interaction_context
-from tarsy.models.unified_interactions import LLMMessage, LLMResponse, LLMChoice
+from tarsy.models.unified_interactions import LLMMessage, LLMConversation, MessageRole
```

Ruff (0.12.2)

Module level import not at top of file

(E402)